docs/content/components/subscribe.mdx

```diff
@@ -36,33 +36,29 @@ https://support.granicus.com/s/article/Using-Subscribe-Field-Snippets
 
 <TabItem displayText="HTML" id="html">
 
-<h3>HTML Example Component</h3>
-
 <Code className="html">{`
 <div class="usa-sign-up">
   <h3 class="usa-sign-up__heading">Sign up for email updates</h3>
   <form
-    action="https://public.govdelivery.com/accounts/{ACCOUNT_ID}/subscribers/qualify"
+    action="https://public.govdelivery.com/accounts/USNIHNCI/subscribers/qualify"
```

Comment:
do we want to be sharing our account id here?

Reply:
I think I am ok with it since it shows up in the rendered HTML. Additionally anyone implementing the subscribe box would use that for any of our subscriptions.